backends/imgui_impl_vulkan.cpp

```diff
@@ -1699,8 +1773,33 @@ static void ImGui_ImplVulkan_CreateWindow(ImGuiViewport* viewport)
     vd->WindowOwned = true;
 
     // Create pipeline (shared by all secondary viewports)
-    if (bd->PipelineForViewports == VK_NULL_HANDLE)
-        ImGui_ImplVulkan_CreatePipeline(v->Device, v->Allocator, VK_NULL_HANDLE, wd->RenderPass, VK_SAMPLE_COUNT_1_BIT, &bd->PipelineForViewports, 0);
+	if (bd->PipelineForViewports == VK_NULL_HANDLE)
+	{
+		ImGui_ImplVulkan_PipelineCreateInfo pci;
+		pci.Device = v->Device;
+		pci.Allocator = v->Allocator;
+		pci.PipelineCache = VK_NULL_HANDLE;
+		pci.RenderPass = wd->RenderPass;
+		pci.Subpass = 0;
+		pci.MSAASamples = VK_SAMPLE_COUNT_1_BIT;
```

Comment:
Might be clean to use a PCI factory so you can one shot this pci struct with something like PCISettings.createBaseVulkanPCI()